packages/react/src/components/Button/Button-story.js

```diff
@@ -75,6 +76,7 @@ const props = {
     }
     return {
       className: 'some-class',
+      expressive: boolean('Expressive', false),
```

Comment:
I keep going back and forth if this prop makes more sense as `expressive` or `isExpressive` to better align throughout the repo...

Reply:
I lean towards `isExpressive` (unless we are anticipating other variants, which would change the prop name)

Reply:
I also lean towards isExpressive. I'll make that change!